content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -0,0 +1,420 @@
+# Pipelines Files (`dvc.yaml`)
+
+You can construct data science or machine learning pipelines by defining
+individual [stages](/doc/command-reference/run) in one or more `dvc.yaml` files
+(or _pipelines files_). Stages form a pipeline when they connect with each other
+(forming a _dependency graph_, see `dvc dag`). Refer to
+[Data Pipelines](/doc/start/data-pipelines).
+
+> Note that a helper command, `dvc run`, is available to create (and execute)
+> stages.
+
+`dvc.yaml` files can be versioned with Git.
+
+These files use the [YAML 1.2](https://yaml.org/) file format, and a
+human-friendly schema explained below. We encourage you to get familiar with it
+so you may modify, write, or generate stages and pipelines on your own.
+
+> Note that we use [GNU/Linux](https://www.gnu.org/software/software.html) in
+> most of our examples.
+
+## Stages
+
+`stages` are named by the user and have a flexible structure. Here's a simple
+example:
+
+```yaml
+stages:
+  print2cols:
+    cmd: awk '{print $1 "\t" $2}' lines.txt > 2columns.txt
+    deps:
+      - lines.txt
+    outs:
+      - 2columns.txt
+```
+
+The most important part of a stage it's the terminal command(s) it executes
+(`cmd` field). This is what DVC runs when the stage is reproduced (see
+`dvc repro`).
+
+If a program executed by the command read input files, they can be defined as
+<abbr>dependencies</abbr> (`deps`). DVC will check whether they have changed to
+decide whether the stage requires re-execution (see `dvc status`).
+
+If the command writes files or directories, they can be defined as
+<abbr>outputs</abbr> (`outs`). DVC will track them going forward (similar to
+using `dvc add`).
+
+### Parameter dependencies
+
+[Parameters](/doc/command-reference/params) are a special type of stage
```

Comment:
There's no non-default case? Was it documented before?

Reply:
> There's no non-default [params] case?

Sorry, Idk what you mean @skshetry.

Params was never explained in the dvc.yaml file before (only mentioned the `params` fields).